scripts/lib/rules/PascalCase.js

```diff
@@ -29,9 +29,19 @@ async function validate({
   publicEventsFunctions,
   onError,
 }) {
+  // Check if extension name is a valid string
+  if (typeof name !== 'string') {
+    onError('Invalid extension name. Expected a string.');
+    return;
+  }
```

Comment:
Typechecking is already statically enforced via typescript: this is unnecessary complexity.